applications/jupyter/workloads.tfvars

```diff
@@ -26,13 +26,13 @@ cluster_membership_id = "" # required for private cluster, defaults to `cluster_
 #######################################################
 
 ## JupyterHub variables
-namespace                     = "ml"
-gcs_bucket                    = "gcs-bucket-dsfhfh"
+namespace         = "ml"
+create_gcs_bucket = true
+gcs_bucket        = "gcs-bucket-dsfhfh"
```

Comment:
should we generate a random suffix to add to the bucket name (can do in a future PR)?